docs/source/contribution.rst

```diff
@@ -123,7 +123,42 @@ It is also appreciated to make such a backport PR by any contributors, though, s
 Issues and Pull Requests
 ------------------------
 
-In this section, we explain how to send pull requests (PRs).
+In this section, we explain how to send issues and pull requests (PRs).
+
+How to File an Issue
+~~~~~~~~~~~~~~~~~~~~
+
+(Work in progress.)
+
+AI Policies
+...........
+
+Recently, CuPy has received an enormous amount of bug reports in which AI/LLM tools were exploited to search
+for, e.g., unexpected corner cases of CuPy in which NumPy's behaviors are mismatched. We would like to reiterate
+that CuPy being a community project is managed and maintained by humans, not bots or AIs, and human beings
+have limited bandwidth and capacity in triaging and responding to bug reports in addition to other development
+activities. In light of recent incidents, we develop the following guidelines:
+
+- Please do not spam CuPy's issue tracker with AI-based contents.
+- Please do not misuse CuPy as a testbed for your AI development or experiment.
+- Please understand :doc:`certain deviations from NumPy <user_guide/difference>` are expected.
+- Please only file bug reports that actually impacts daily/production use of CuPy, not for hypothetical/pedantic
+  scenarios.
+- Please make sure bug reports are *not filed anonymously* (see below).
+
+The CuPy team reserves rights to close any bug reports proactively and preemptively without further notices, if
+it is determined by the team that the reports are potentially filed with computer programs, including AI/LLM tools.
+A typical alerting sign of misusing AI tools is massive bug reports filed by the same GitHub account that has all
+activities locked (which we referred to as "anonymously"), as shown with GH profiles that look like this:
+
+.. image:: ./_static/anonymous_github_profile.png
+   :width: 300
+
+which gives the team no way to determine the true intention or potential use cases.
+
+If you believe your bug reports are legitimate and can be encountered in real life, but get closed by accident,
+please kindly let us know via the `mailing list <https://groups.google.com/forum/#!forum/cupy>`_.
```

Comment:
You could have folks just reopen with a comment to that they were not generated by AI.

I don't know if you have this policy but I'm tempted to start banning users on some of my projects that just use rag systems to make non-material changes. It is infuriating.

Reply:
Thanks, Andy! Yes, it is infuriating no doubt... Luckily far we haven't seen such PRs (@kmaehashi would know the best) and I can extend the latter section ("how to file a PR") on this if needed. Regarding reopening a closed issue, I am under the impression that it involves access control and not every user has permission to reopen. I could be wrong, though!

Reply:
For now we are not seeing such PRs, 

Self-reopening issues can only be done if they are closed by themselves; otherwise triage permission is required.

Another idea would be to ask them to mention the team `@cupy/triage-team` to get the attention and request for reopen. As we don't have a maintainers' mailing list, and the current list is primarily intended for users, I think it's better not to use the list for this purpose.

Reply:
> Another idea would be to ask them to mention the team `@cupy/triage-team` to get the attention and request for reopen. As we don't have a maintainers' mailing list, and the current list is primarily intended for users, I think it's better not to use the list for this purpose.

I will make change on this. Though I have a feeling that pinging the triage team is not easy to teach... The easiest path might be to set up a bot leaving a message on issues closed without resolution: "To reopen this issue, please ping `@cupy/triage-team`".

Reply:
Addressed in commit fd280f26288d7e30ff7c7a9fe3cb413392b9d14d.